cpp/unit/ability.h

```diff
@@ -42,17 +43,14 @@ using ability_set = std::bitset<16>;
 using ability_id_t = unsigned int;
 
 /**
- * all bits set to 1
+ * set bits corresponding to abilities
  */
-const ability_set ability_all = ability_set().set();
+ability_set from_list(const std::vector<ability_type> &items);
```

Comment:
so, there's a method `::openage::from_list` which returns `std::bitset<16>`... maybe the method should have a bit more... `ability_set`-specific name. like `to_ability_set`